samples/samples-js/AddProductParams/function.json

```diff
@@ -0,0 +1,28 @@
+{
+  "bindings": [
+    {
+      "authLevel": "anonymous",
+      "name": "req",
+      "direction": "in",
+      "type": "httpTrigger",
+      "methods": [
+        "get",
+        "post"
+      ],
+      "route": "addproduct/{productid}/{name}/{cost}"
```

Comment:
The route should generally match the function name for our samples. And this isn't how you define the route - the query parameters for GET requests aren't included as part of this (what you have here is part of the path itself, e.g. `addproduct/1/MyItem/100`, parameters would look like `addproduct?id=1&Name=MyItem&Cost=100`)